sale_stock_invoice_lot_quantity/views/report_invoice.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="custom_stock_account_report_invoice_document"
+        inherit_id="stock_account.stock_account_report_invoice_document"
+        priority="100"
+    >
+        <xpath expr="//t[contains(@t-esc, 'product_name')]" position="replace">
+            <span>
+                <t t-esc="snln_line['product_name']" />
+            </span>
+        </xpath>
+        <xpath expr="//t[contains(@t-esc, 'quantity')]" position="replace">
+            <span t-field="snln_line['lot_quantity'].quantity" />
+        </xpath>
+        <xpath expr="//t[contains(@t-esc, 'uom_name')]" position="replace">
+            <span>
+                <t t-esc="snln_line['uom_name']" groups="uom.group_uom" />
+            </span>
+        </xpath>
+        <xpath expr="//t[contains(@t-esc, 'lot_name')]" position="replace">
+            <span>
+                <t t-esc="snln_line['lot_name']" />
+            </span>
+        </xpath>
```

Comment:
@AungKoKoLin1997 Wouldn't it make more sense to assign `display:none` to the `tr` tag and define new `tr` instead of doing these individual replaces?

Reply:
Yes. I agree with you. Updated the code.